build/testIntegration.sh

```diff
@@ -51,6 +51,13 @@ fi
 
 echo
 
+if [ -n "$2" ]; then
+    echo "Setting environment variable 'ORYX_TEST_IMAGE_BASE' to provided value '$2'."
+    export ORYX_TEST_IMAGE_BASE="$2"
```

Comment:
Change to `ORYX_TEST_REGISTRY_NAME`?

Reply:
what about the other environment variable here?

Reply:
Changed this environment variable.